contracts/src/test/kotlin/com/r3/developers/apples/contracts/ApplesContractTest.kt

```diff
@@ -8,9 +8,32 @@ import net.corda.v5.ledger.utxo.StateAndRef
 import java.security.PublicKey
 import java.util.UUID
 
+/**
+ * The following is the base implementation of the Contract Tests for the Apples CSDE-template-tutorial.
+ *
+ * - The AppleContractTest abstract class implements the ContractTest class.
+ * - For full contract test coverage, we generally create a class for every command scenario for every state.
+ * - Each of these classes will implement the abstract class to incorporate ContractTest general testing functionality
+ *   as well as the functionality specific for this CordApp tutorial example.
+ *
+ * In this case, we have 3 scenarios (you can refer to contract files for the apples tutorial):
+ *      1. AppleStamp state, AppleCommands.Issue command
+ *      2. BasketOfApples state, AppleCommand.PackBasket command
+ *      3. BasketOfApples state, AppleCommand.Redeem command
+ *
+ * The variables and methods within this abstract class should to written to enable the re-use of code such that states
```

Comment:
The variables and methods within this abstract class are written to enable code re-use such that states